graphql-schema/schema.graphql

```diff
@@ -127,6 +127,7 @@ type User {
     sex: String
     status: String
     groups: [Group]
+    past_groups: [PastGroups] @hasMany(relation: "pastGroups")
```

Comment:
`past_groups: [Group]  @hasMany(relation: "pastGroups")`
Might be a better/simpler option.